### PR TITLE
#22: Updated version to 0.32.0, added stack size

### LINKED ISSUE #22: Update eolang.version to v0.32.0
#9 fails, so we need to do it manually

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #22 
I added `stack-size` because rultor cannot merge it due to stack-overflow.
Codecov was already failed in #9 .

---

## PR-Codex overview
This PR focuses on updating the `codecov-action` version and adding configuration to the `maven-surefire-plugin`. 

### Detailed summary
- Updated `codecov-action` version from `v4` to `v4-beta`.
- Added configuration to the `maven-surefire-plugin` in `pom.xml`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by levBagryansky** [comment]
I use v4-beta according to https://github.com/codecov/codecov-action/issues/1089 and https://github.com/codecov/codecov-action/issues/1091

**Comment by levBagryansky** [comment]
@Graur please take a look

File: pom.xml [hunk 1]
```diff
@@ -104,6 +108,21 @@ SOFTWARE.
...
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>
+            @{argLine} -Xss${stack-size}
+          </argLine>
+          <properties>
+            <configurationParameters>
+              junit.jupiter.execution.parallel.enabled = true
+              junit.jupiter.execution.parallel.mode.default = concurrent
```
Comment (Graur) [inline pom.xml:120]: @levBagryansky Do we really need to execute this project's tests in paralell?

**Review by Graur — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Graur) [reply pom.xml:120]: @levBagryansky Do you mean 8 minutes without parallell execution?

Comment (levBagryansky) [reply pom.xml:120]: @Graur yes. Also it is already enabled in eo repository

Comment (Graur) [reply pom.xml:120]: @levBagryansky Did you try to run it without concurrent execution?

Comment (levBagryansky) [reply pom.xml:120]: @Graur I measured again and parallel testing takes `08:22` min but sequential testing takes `10:20 min.`

**Review by Graur — APPROVED** [review]

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**